torchtitan/models/deepseek_v3/infra/parallelize.py

```diff
@@ -54,7 +55,7 @@ def parallelize_deepseekv3(
         apply_non_moe_tp(
```

Comment:
Need to add the same check  `job_config.training.seq_len % parallel_dims.seq_len_divisor` for TP here.  You could add it or I could add it in next PR